### PR TITLE
Traffic Mirroring Switch

### LINKED ISSUE #65: SDN Mirror
Write a pox program that acts as a learning switch, but also mirrors all traffic from a specified IP address (first command line argument) to another ethernet port (second command line argument).  Use tcpdump or the packetsniffer to verify that the second host gets the traffic (but that other hosts do not).

### PR DESCRIPTION
**Pull request by phil-lopreiato**
It works. Probably?

Closes #65.

### CODE REVIEW

File: sdn/pox/ext/sdn-mirror.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+
+from pox.core import core
+import pox.openflow.libopenflow_01 as of
+from pox.openflow import *
+from utils import *
+from SimpleL2Learning import SimpleL2LearningSwitch
+
+log = core.getLogger()  # Use central logging service
+
```
Comment (twood02) [inline sdn/pox/ext/sdn-mirror.py:15]: Write me some comments up top to explain what this does.

**Comment by phil-lopreiato** [comment]
Comments added. Should be good to go

**Comment by twood02** [comment]
does it still "probably" work or has it been tested on GENI or mini-net?

**Comment by phil-lopreiato** [comment]
It's been tested, so it works even better than "probably"

As another note, when you imported the BBN modules, you shouldn't have used a dash in the names, since you [can't](http://stackoverflow.com/questions/7583652/python-module-with-a-dash-or-hyphen-in-its-name) use the regular import. In 5887114, I renamed the file I extended to use an underscore instead, which will work. 

I can update the rest of the files to follow convention, if you'd like

**Comment by twood02** [comment]
please make a PR with the file names changed.  thanks!

---

Timothy Wood, Ph. D.
Assistant Professor
Department of Computer Science
The George Washington University
http://www.seas.gwu.edu/~timwood

**Pull request merged**